app/helpers/color_helper.rb

```diff
@@ -0,0 +1,10 @@
+module ColorHelper
+  def gradient_from_hex(hex = "#4F46E5")
```

Comment:
`Settings::UserExperience` already has a default color of `#3b49df`. Is this fallback here necessary? I don't have a strong opinion.

Reply:
Oh sure, this is an interesting piece that I should probably add a comment to. The gist of it is that the requested default values from the issue were indigo-600 and indigo-900 (based on darkenings of the default user experience color), and when I put the rgb of indigo-600 into a converter, #4f46e5 is what came out.